sonic_platform_base/sonic_xcvr/sfp_optoe_base.py

```diff
@@ -55,6 +55,10 @@ def is_coherent_module(self):
         api = self.get_xcvr_api()
         return api.is_coherent_module() if api is not None else None
 
+    def is_transceiver_vdm_supported(self):
+        api = self.get_xcvr_api()
+        return api.is_transceiver_vdm_supported() if api is not None else None
```

Comment:
return False on falure? @mihirpat1

Reply:
@prgeor I am currently returning None because the caller (`xcvrd` in this case) treats None as False and will not perform VDM-related operations. This approach ensures that the current behavior remains unchanged. Additionally, if in the future the caller needs to differentiate between False (explicitly not supported) and None (indeterminate or error state), the current implementation will allow for this distinction without requiring changes to the existing implementation.